gui/default/syncthing/folder/editFolderModalView.html

```diff
@@ -132,9 +132,9 @@
               <span translate ng-if="folderEditor._guiVersioning.staggeredMaxAge.$error.min && folderEditor._guiVersioning.staggeredMaxAge.$dirty">A negative number of days doesn't make sense.</span>
             </p>
           </div>
-          <div class="form-group" ng-if="currentFolder._guiVersioning.selector == 'staggered'">
-            <label translate for="staggeredVersionsPath">Versions Path</label>
-            <input name="staggeredVersionsPath" id="staggeredVersionsPath" class="form-control" type="text" ng-model="currentFolder._guiVersioning.staggeredVersionsPath" />
+          <div class="form-group" ng-if="currentFolder._guiVersioning.selector != 'none' && currentFolder._guiVersioning.selector != 'external'">
```

Comment:
Just a suggestion, I am fine with how it is now as well:  
I'd put this into some utility function on `$scope` and also use that to set it once when initialising and saving the folder.

Reply:
I have tried to fiddle around with the code, but my JS skills seem to be too limited for this, unfortunately :sweat:. What do you mean exactly by "set it once when initialising and saving the folder"?

Reply:
Don't worry about it, lets merge as is. Doing a light cleanup on the folder edit JS is on my low-prio-to-do anyway - I'll make it medium-prio then :)